_gsocproposals/2019/proposal_ActsAlign.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Improving the precision of algorithms to hunt particles
+layout: gsoc_proposal
+project: Acts, IRIS-HEP
+year: 2019
+organization:
+- CERN
+- UC Berkeley
+- LBNL
```

Comment:
Could you please copy the organization description for `LBNL` from [`_gsocorgs/2018/lbnl.md`](https://github.com/HSF/hsf.github.io/blob/master/_gsocorgs/2018/lbnl.md) to `_gsocorgs/2019`?

Reply:
Done